python/paddle/fluid/tests/unittests/ir/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Are these two commented files needed?

Reply:
Yes they allow to do the following import:  
> from ir.inference.inference_pass_test import InferencePassTest
in the `test_matmul_op_output_fused.py